insights/specs/default.py

```diff
@@ -316,6 +316,7 @@ def dumpdev(broker):
     fc_match = simple_command("/bin/fc-match -sv 'sans:regular:roman' family fontformat")
     fcoeadm_i = simple_command("/usr/sbin/fcoeadm -i")
     fdisk_l = simple_command("/sbin/fdisk -l")
+    findmnt_lo_propagation = simple_command("/usr/bin/findmnt -lo+PROPAGATION")
```

Comment:
You should change the path of the command to `/bin/findmnt` instead of `/usr/bin/findmnt` to ensure compatible with all RHEL 6/7/8. `/usr/bin/findmnt` is incorrect in RHEL6

Reply:
Done